app/controllers/api/v2/aliquots_controller.rb

```diff
@@ -5,6 +5,8 @@ module V2
     # Provides a JSON API controller for aliquot
     # See: http://jsonapi-resources.com/ for JSONAPI::Resource documentation
     class AliquotsController < JSONAPI::ResourceController
+      include Api::V2::Concerns::ApiKeyAuthenticatable
```

Comment:
If this is required for all v2 resources and all future resources? you could include it in the JSON API initializer or have it inherited from somewhere.